src/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "../tsconfig.base",
   "compilerOptions": {
     "rootDir": ".",
-    "outDir": "../lib"
+    "outDir": "../lib",
+    "importHelpers": false
```

Comment:
Or maybe add `tslib` as dep instead?

Reply:
Of course, but I think this can be handled when compiling, and there is no need to add runtime packages. This is why I suggest using packaging tools, which can handle these problems better and can also generate independent entries for esm and cjs.

Reply:
Sharable deps would reduce the package size. There is no need to add packaging tools in this plugin IMO.

Reply:
I don't think this is a reason.

In my test, the size of all compiled js files under the current configuration is about **314743 bytes**. After adding `"importHelpers": false`, the size is about **317433 bytes**, an increase of only 0.85%. After packaging with tsup, the size is about **274985 bytes**, a reduction of 12%.

Therefore, if you consider the package size, the best approach should be to use a packaging tool to further reduce the size. Even if the packaging tool is not considered, `tslib` should not be introduced. The size of `tslib` itself is much larger than the 0.85% increment.

Of course, I am just making my suggestions, the final decision still rests with you.

Reply:
Please open a new issue instead.